mantidproject/mslice
# Refactor figure manager

Refactored figure manager which keeps track of current and kept figures. It should now be more robust and should make getting a working cli interface easier.

**To test:**
Start mslice and verify that keep and make current behave as expected.

Fixes #56
